go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/PuerkitoBio/purell v1.1.1
 	github.com/agrea/ptr v0.0.0-20180711073057-77a518d99b7b
 	github.com/cenkalti/backoff/v4 v4.1.3
-	github.com/dapr/components-contrib v1.8.0-rc.1.0.20220808165304-1148776d370d
+	github.com/dapr/components-contrib v1.8.0-rc.1.0.20220810215246-f2ac0cda69ec
```

Comment:
On reason for updating components-contrib: needed a fix in the cron component for the E2E tests